third_party/gpus/cuda_configure.bzl

```diff
@@ -0,0 +1,248 @@
+# -*- Python -*-
+
+load(
+    "@bazel_tools//tools/cpp:configure_util.bzl",
+    "find_cc",
+    "get_cxx_inc_directories",
+)
+
+
+_DEFAULT_CUDA_TOOLKIT_PATH = "/usr/local/cuda"
+_DEFAULT_CUDNN_INSTALL_PATH = "/usr/local/cuda"
+
+
+def _cuda_toolkit_path(repository_ctx):
+  """Finds the cuda toolkit directory."""
+  cuda_toolkit_path = _DEFAULT_CUDA_TOOLKIT_PATH
+  if "CUDA_TOOLKIT_PATH" in repository_ctx.os.environ:
```

Comment:
Params that were previously configured using the `./configure` script prompts are now being passed using environment variables. For example:

```sh
CUDA_VERSION=7.5 bazel test --config=cuda //tensorflow/...
```

The problem with this is that changing the environment variables would not automatically trigger a re-fetch of the `@local_config_cuda` external repository.

Is there a way to pass these values using command line flags and apply them using Bazel's configuration mechanism? I'm not sure how to do this with `config_setting` since it seems to only match specific values.